nilearn/image/resampling.py

```diff
@@ -118,7 +118,7 @@ def coord_transform(x, y, z, affine):
     --------
     Transform data from coordinates to brain space. The "affine" matrix
     can be found as the ".affine" attribute of a nifti image, or using
-    the "get_affine()" method for older nibabel installations::
+    the "get_affine()" method for older nibabel installations:
```

Comment:
No: that was correct. "::" is a special marker in restructured text that starts a verbatim block (or code block).

Reply:
thanks for the explanation! Reverted back.